.baedeker/up.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 set -e
 BDK_DIR=$(dirname $(readlink -f "$0"))
-RUST_LOG=info baedeker --spec=docker -J$BDK_DIR/vendor/ --generator=docker_compose=$BDK_DIR/.bdk-env --generator=docker_compose_discover=$BDK_DIR/.bdk-env/discover.env --secret=file=$BDK_DIR/.bdk-env/secret  --tla-str=relay_spec=rococo-local --input-modules='lib:baedeker-library/ops/nginx.libsonnet' --input-modules='lib:baedeker-library/ops/devtools.libsonnet' --tla-str=repoDir=$(realpath $BDK_DIR/..) $@ $BDK_DIR/rewrites.jsonnet
+RUST_LOG=info ./baedeker --spec=docker -J$BDK_DIR/vendor/ --generator=docker_compose=$BDK_DIR/.bdk-env --generator=docker_compose_discover=$BDK_DIR/.bdk-env/discover.env --secret=file=$BDK_DIR/.bdk-env/secret  --tla-str=relay_spec=rococo-local --tla-code=assethub_spec="import 'assethub-spec.json'" --input-modules='lib:baedeker-library/ops/nginx.libsonnet' --input-modules='lib:baedeker-library/ops/devtools.libsonnet' --tla-str=repoDir=$(realpath $BDK_DIR/..) $@ $BDK_DIR/rewrites.jsonnet
```

Comment:
```suggestion
RUST_LOG=info baedeker --spec=docker -J$BDK_DIR/vendor/ --generator=docker_compose=$BDK_DIR/.bdk-env --generator=docker_compose_discover=$BDK_DIR/.bdk-env/discover.env --secret=file=$BDK_DIR/.bdk-env/secret  --tla-str=relay_spec=rococo-local --tla-code=assethub_spec="import 'assethub-spec.json'" --input-modules='lib:baedeker-library/ops/nginx.libsonnet' --input-modules='lib:baedeker-library/ops/devtools.libsonnet' --tla-str=repoDir=$(realpath $BDK_DIR/..) $@ $BDK_DIR/rewrites.jsonnet
```

Зачем его не из PATH?

Reply:
Согласен! нужно запускать из PATH. 
у меня ещё вопрос: нужно ли вот это прописывать здесь --tla-code=assethub_spec="import 'assethub-spec.json'". Запуск Ассетхаба - это частный случай.

Reply:
> Зачем его не из PATH?

Случайно попало в коммит. Я всегда у себя в up.sh заменяю `baedeker` на `./baedeker`, так как не храню его в PATH.

Может быть, в идеале было бы сделать что-то такое:

```bash
if [ -x "./baedeker" ]; then
	./baedeker bla-bla
elif [ -x "$(command -v baedeker)" ]; then
	baedeker bla-bla
else
	echo "Baedeker is not installed"
fi
``` 

> Запуск Ассетхаба - это частный случай

Это удобно для запуска xcm*.jsonnet. Давайте оставим